x-pack/plugins/maps/public/index_pattern_util.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IFieldType, IndexPattern } from 'src/plugins/data/public';
+import { i18n } from '@kbn/i18n';
+import { getIndexPatternService, getIsGoldPlus } from './kibana_services';
+import { indexPatterns } from '../../../../src/plugins/data/public';
+import { ES_GEO_FIELD_TYPE, ES_GEO_FIELD_TYPES } from '../common/constants';
+
+export function getGeoTileAggNotSupportedReason(field: IFieldType): string | null {
+  if (!field.aggregatable) {
+    return i18n.translate('xpack.maps.geoTileAgg.disabled.docValues', {
+      defaultMessage: 'Clustering requires doc_values.',
```

Comment:
I'm not sure users will know what `doc_values` are.  Maybe something like `This geo_shape field does not have doc_values. Perhaps the index was created before version 7.8?`.

Reply:
Don't want to get too technical in error message so maybe not calling out version numbers.

How about "Clustering requires aggregations. Enable aggregations with doc_values."? I figured we don't want to get too technical with error message in the app but may be add a section to our trouble shooting guide that explains what doc_values are and why they may not exist for geo_shape index

Reply:
👍